platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -96,19 +98,20 @@ public ImportMigrationManagerImpl(
           migratables.collect(
               Collectors.toMap(
                   Migratable::getId,
-                  m -> new ImportMigrationContextImpl(report, zip, m),
+                  m -> new ImportMigrationContextImpl(report, zipContents, m),
                   ConfigurationMigrationManager.throwingMerger(),
                   LinkedHashMap::new)); // to preserved ranking order and remove duplicates
       // add a system contexts
-      contexts.put(null, new ImportMigrationContextImpl(report, zip));
-      zip.stream()
-          .filter(ze -> !ze.isDirectory())
-          .map(ze -> new ImportMigrationEntryImpl(this::getContextFor, ze))
+      contexts.put(null, new ImportMigrationContextImpl(report, zipContents));
+      zipContents
+          .entrySet()
+          .stream()
+          .filter(k -> !k.getKey().isDirectory())
```

Comment:
❗️ might no longer be needed to filter here as we should filter when we actually go through the content of the zip such that we don't try to read the content (empty) of directory entries. I would suggest you retest the directory filtering by taking the zip, decrypting it, unzipping it, re-zipping from the command-line which will introduce directory entries and then re-encrypting it.

Reply:
Yeah that would make more sense, I'll move the filtering out to the `ZipUtils` class instead